src/Symfony/Component/Console/Application.php

```diff
@@ -119,9 +120,15 @@ public function run(InputInterface $input = null, OutputInterface $output = null
 
         $this->configureIO($input, $output);
 
+        unset($e);
```

Comment:
I figured, that since the try block is not the first thing in the function, someone might add more code here eventually and forget that there is an `isset($e)`... another, maybe nicer solution to that paranoia might be changing the name of the exception variable?